app/assets/stylesheets/modules/profile.scss

```diff
@@ -0,0 +1,13 @@
+
+.circular img {
+  height:200px;
```

Comment:
Properties should be ordered border-radius, height, margin, transform, transition, width<br>Colon after property should be followed by one space